docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +750,41 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+------------------
```

Comment:
@mosabua do we expect the header section underline to be same length in characters as the header section title itself?

Reply:
yes